Common/UnitDefinitions/Mass.json

```diff
@@ -136,6 +136,12 @@
       "BaseUnits": {
         "M": "Ounce"
       },
+      "Conversions": [
```

Comment:
`DirectConversions` maybe, to be more explicit that this is an optional thing?

Reply:
@angularsen I think I would actually make it mandatory. I may take in stages, but maybe the first stage is the convert the current FromUnitToBaseFunc/FromBaseToUnitFunc to the Conversions?

Reply:
Oh gotcha, yes I suppose we could refactor the existing definitions to this.

The upside is more explicit naming, the old naming is kinda confusing. Also gathering all conversion functions, both direct and via base, in one place might be better.

The downside is that it is maybe not as obvious that the conversions need to go via a "base" unit, but we could maybe fail codegen if any unit has missing conversions from/to base unit?